test/browser/sauce-browsers.json

```diff
@@ -1,38 +1,90 @@
 [
   {
-    "browserName": "firefox",
-    "platform": "WIN7"
+    "browserName": "internet explorer",
+    "platform": "XP",
+    "version": "7"
+  }, {
+    "browserName": "internet explorer",
+    "platform": "WIN7",
+    "version": "8"
+  }, {
+    "browserName": "internet explorer",
+    "platform": "WIN7",
+    "version": "9"
+  }, {
+    "browserName": "internet explorer",
+    "platform": "WIN8",
+    "version": "10"
+  }, {
+    "browserName": "internet explorer",
+    "platform": "WIN8.1",
+    "version": "11"
   },
+
+
   {
     "browserName": "firefox",
-    "platform": "WINXP",
-    "browserVersion": "3"
-  },
-  {
-    "browserName": "chrome",
-    "platform": "WIN7"
+    "platform": "WIN8",
+    "version": "3.6"
+  }, {
+    "browserName": "firefox",
+    "platform": "WIN8",
+    "version": "35"
+  }, {
+    "browserName": "firefox",
+    "platform": "WIN8",
+    "version": "36"
+  }, {
+    "browserName": "firefox",
+    "platform": "WIN8",
+    "version": "37"
+  }, {
+    "browserName": "firefox",
+    "version": "beta"
   },
+
+
   {
-    "browserName": "internet explorer",
-    "platform": "WIN8"
+    "browserName": "opera",
+    "platform": "Linux",
+    "version": "12"
   },
+
+
   {
-    "browserName": "safari",
-    "platform": "OS X 10.6",
-    "version": "5"
+    "browserName": "chrome",
+    "platform": "WIN8",
+    "version": "39"
```

Comment:
Shame there is no way to say latest stable + the previous versions. Guess we'll just need to keep on top of this.